sdks/java/managed/src/main/java/org/apache/beam/sdk/managed/Managed.java

```diff
@@ -124,7 +129,9 @@ public static ManagedTransform read(String source) {
    * managed sinks are:
    *
    * <ul>
-   *   <li>{@link Managed#ICEBERG} : Write to Apache Iceberg
+   *   <li>{@link Managed#ICEBERG} : Write to Apache Iceberg tables
+   *   <li>{@link Managed#KAFKA} : Write to Apache Kafka topics
+   *   <li>{@link Managed#BIGQUERY} : Write to GCP BigQuery tables
```

Comment:
Probably also mention that read/write methods will be determined dynamically.

Reply:
This sounds like implementation detail -- does the user need to know this?

Reply:
Yeah, may be we can mention something in the Website documentation